imagej/__init__.py

```diff
@@ -792,6 +825,24 @@ def _java_to_dataset(self, data):
                 raise exc
             raise TypeError('Cannot convert to dataset: ' + str(type(data)))
 
+        def _java_to_img(self, data):
+            """
+            Converts the data into a ImageJ Img
+            """
+            # This try checking is necessary because the set of ImageJ converters is not complete.
+            try:
+                if self._ij.convert().supports(data, Img):
+                    img = self._ij.convert().convert(data, Img)
+                    return self._ij.dataset().create(ImgPlus(img))
```

Comment:
This is not converting to `Img`, it's converting to `Dataset`. So I think the naming of the functions is not correct.

Reply:
Oh, good catch!

Should be fixed with 85130da